app/nation/app/Router.js

```diff
@@ -4494,4 +4494,4 @@ $(function() {
             });
         }
     }))
-})
+})
```

Comment:
it kinda looks like that the only maybe valid fix here is the whitespaces one in line 3707 ...
or in other words theoretically there is no change at all

@razu9861 think we have to check your editors whitespace policy